### PR TITLE
Fix eth vevrsion alert

### PR DESCRIPTION
**Pull request by jingweicb**
Fixes # .

### Solution
import the ethereum repo

### CODE REVIEW

**Comment by jingweicb** [comment]
the previous breaking is because https://github.com/coinbase/rosetta-sdk-go/runs/7793099183?check_suite_focus=true 
what I do is 
1 run go get github.com/ethereum/go-ethereum 
2 go mod tiy 
3 make check-gen
@shrimalmadhur

File: go.mod [hunk 1]
```diff
@@ -36,6 +36,7 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/klauspost/compress v1.12.3 // indirect
+	github.com/kr/pretty v0.2.1 // indirect
```
Comment (shrimalmadhur) [inline go.mod:39]: wait this is not eth?

Comment (jingweicb) [reply go.mod:39]: not, but I see this add into here when run  go get github.com/ethereum/go-ethereum

Comment (shrimalmadhur) [reply go.mod:39]: also the latest is `v1.10.21` I think the problem is latest has some issues. check out line 12 - we use latest. so we can't do anything for now

**Pull request closed without merging**